dbt/adapters/databricks/python_models/python_submissions.py

```diff
@@ -162,3 +162,161 @@ def submit(self, compiled_code: str) -> None:
 class ServerlessClusterPythonJobHelper(BaseDatabricksHelper):
     def submit(self, compiled_code: str) -> None:
         self._submit_through_notebook(compiled_code, {})
+
+
+class WorkflowPythonJobHelper(BaseDatabricksHelper):
+
+    @property
+    def default_job_name(self) -> str:
+        return f"{self.database}-{self.schema}-{self.identifier}__dbt"
+
+    @property
+    def notebook_path(self) -> str:
+        return f"{self.notebook_dir}/{self.identifier}"
+
+    @property
+    def notebook_dir(self) -> str:
+        return f"/Shared/dbt_python_model/{self.database}/{self.schema}"
```

Comment:
We should probably use the folder api logic.  I've been told that using Shared is an anti-pattern and that we might get rid of it as we continue with improved governance.

Reply:
updated to use the new API classes